### DISCUSSION TITLE
wire:click and wire:loading don't work together on the same element

### DISCUSSION THREAD

**Discussion by stuartcusackie** [livewire/livewire#5622] [7. Bugs (archived)]
On Livewire 2.12.3.

**This doesn't work (element not hiding):** 
```
<button
  type="button"
  wire:loading.remove
  wire:click="$set('q', '')"
  >
  <i class="fal fa-times"></i>
</button>

```
**But this does:**
```
<div wire:loading.remove>
  <button
  type="button"
  wire:click="$set('q', '')"
  >
    <i class="fal fa-times"></i>
  </button>
</div>
```

Thanks!

**Comment by alekmlynek** [comment]
I've run into the same issue, and I don't know if this has been addressed.

`<button wire:click="addToCartCallback" wire:loading.attr='disabled' type="button" class="ml-6 px-8 py-3 border border-transparent text-xs sm:text-sm font-medium w-full rounded-2xl shadow-sm text-white bg-blue-600 disabled:bg-gray-600">`

Does not work. It does work if I remove the wire:click event.
